devtools/gradle/src/main/java/io/quarkus/gradle/builder/QuarkusModelBuilder.java

```diff
@@ -48,17 +46,27 @@
 import io.quarkus.gradle.tasks.QuarkusGradleUtils;
 import io.quarkus.runtime.LaunchMode;
 
-public class QuarkusModelBuilder implements ParameterizedToolingModelBuilder {
+public class QuarkusModelBuilder implements ParameterizedToolingModelBuilder<ModelParameter> {
 
-    private static final List<String> scannedConfigurations = new LinkedList();
```

Comment:
This static field is in fact critical a critical issue because an instance of this class will be shared between builds (and probably also projects).